.travis.yml

```diff
@@ -7,13 +7,18 @@ go:
   - 1.6
 
 env:
-  - TOOLS_CMD=golang.org/x/tools/cmd
+  global:
+    - TOOLS_CMD=golang.org/x/tools/cmd
+    - PATH=$GOROOT/bin:$PATH
 
 install:
  - go get ${TOOLS_CMD}/vet
+ - go get ${TOOLS_CMD}/cover
+ - go get github.com/modocache/gover
+ - go get github.com/mattn/goveralls
 
 script:
- - sudo -E /bin/bash -c 'PATH=$GOROOT/bin:$PATH ./test'
```

Comment:
@steveeJ - I agree. The issue we seem to be having is due to the files not being found which makes me think something is up with the `GOPATH`. Can we try putting it back and see if it works properly.